core/state/statedb.go

```diff
@@ -368,6 +368,38 @@ func (s *StateDB) HasSelfDestructed(addr common.Address) bool {
 	return false
 }
 
+// CheckTransactionConditional validates the account preconditions against the statedb.
+//
+// NOTE: A lock is not held on the db while the conditional is checked. The caller must
+// ensure no state changes occur while this check is executed.
+func (s *StateDB) CheckTransactionConditional(cond *types.TransactionConditional) error {
+	cost := cond.Cost()
+	if cost > types.TransactionConditionalMaxCost {
```

Comment:
Does the cli flag modify this global `types.TransactionConditionalMaxCost` ?

Reply:
it doesn't. could make it configurable but decided not to since the spec described 1k and it's other implementations hardcoded the same.

do you think it's worth making this configurable?

Reply:
I think its fine to not make configurable